core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.component.html

```diff
@@ -1,19 +1,23 @@
 <div class="modal-header create-new-story-header">
   <h3>
-    New Story
+    New story
   </h3>
+  <i class="fa fa-times modal-close-button" (click)="cancel()"></i>
 </div>
-<form ng-submit="confirm(story)">
+<form (submit)="save()">
   <div class="modal-body create-new-story">
     <div class="story-input">
       <div class="story-input-header">
         <span>Title*</span>
       </div>
-      <input class="form-control protractor-test-new-story-title-field required" ng-model="story.title" maxlength="<[MAX_CHARS_IN_STORY_TITLE]>" aria-label="Title" autofocus>
+      <input class="form-control protractor-test-new-story-title-field required"
+             [(ngModel)]="newlyCreatedStory.title"
+             [maxlength]="MAX_CHARS_IN_STORY_TITLE"
+             [ngModelOptions]="{standalone: true}" autofocus>
       <div>
         <span class="oppia-input-box-subtitle">
           <em>
-            Story title should be at most <[MAX_CHARS_IN_STORY_TITLE]> characters.
+           Story title should be at most {{MAX_CHARS_IN_STORY_TITLE}} characters.
```

Comment:
Please add space after/before interpolation symbols (`{{` and `}}`)

Reply:
Done